### ISSUE TITLE
improve 'previous' and 'next' alt text

### ISSUE THREAD

**Issue by Smullz622** [psu-libraries/psulib_blacklight#1393]
The 'previous' and 'next' links at the top of the search results page would make more sense for a screen reader if the alt text read 'previous result' and 'next result', especially on the individual result page.
